docs/src/components/ComponentDoc/ComponentProp/ComponentPropFunctionSignature.tsx

```diff
@@ -22,7 +31,7 @@ const rowStyle: any = {
 
 const getTagType = tag => (tag.type.type === 'AllLiteral' ? 'any' : tag.type.name)
 
-const ComponentPropFunctionSignature: any = ({ name, tags }) => {
+const ComponentPropFunctionSignature = ({ name, tags }: ComponentPropFunctionSignature) => {
```

Comment:
maybe `ComponentPropFunctionArg(s)` would be better?

Reply:
done, thanks